pkg/gitreceive/build.go

```diff
@@ -167,6 +167,7 @@ func build(
 			conf.PodNamespace,
 			appConf.Values,
 			slugBuilderInfo.TarKey(),
+			gitSha.Short(),
```

Comment:
The env is added to the dockerBuilderPod as well, it might be useful for someone?